src/docfx/lib/git/GitUtility.cs

```diff
@@ -128,9 +129,14 @@ private static Task<string> ExecuteQuery(string cwd, string commandLineArgs, Tim
         {
             Debug.Assert(!string.IsNullOrEmpty(cwd));
 
-            // todo: check git exist or not
-            var response = await ProcessUtility.Execute("git", commandLineArgs, cwd, timeout, outputHandler);
-            return parser(response);
+            try
+            {
+                return parser(await ProcessUtility.Execute("git", commandLineArgs, cwd, timeout, outputHandler));
+            }
+            catch (Win32Exception ex) when (ProcessUtility.IsNotFound(ex))
```

Comment:
>Win32Exception [](start = 19, length = 14)

this is not cross platform?

Reply:
this **is** xplat, linux test passed. 

.Net core cannot make breaking changes like this (renaming exception), so it'll still throw Win32Exception on linux and mac.

Reply:
looks good!

---
In reply to: [200887867](https://github.com/dotnet/docfx/pull/3022#discussion_r200887867) [](ancestors = 200887867)